test/weights.jl

```diff
@@ -36,6 +36,22 @@ weight_funcs = (weights, aweights, fweights, pweights)
     @test sum(sa, wv) === 7.0
 end
 
+# isequal
+
+@testset "isequal" begin
+
```

Comment:
Remove empty line. It would make sense to test with `NaN` or `-0.0`, since that's what makes `isequal` different from `==`.